paymentsheet/dependencies/dependencies.txt

```diff
@@ -375,15 +385,16 @@
 |    |    |         |         |    \--- androidx.compose.animation:animation-core-android:1.6.8
 |    |    |         |         |         +--- androidx.annotation:annotation:1.1.0 -> 1.8.0 (*)
 |    |    |         |         |         +--- androidx.collection:collection:1.4.0 (*)
-|    |    |         |         |         +--- androidx.compose.runtime:runtime:1.6.8
-|    |    |         |         |         |    \--- androidx.compose.runtime:runtime-android:1.6.8
+|    |    |         |         |         +--- androidx.compose.runtime:runtime:1.6.8 -> 1.7.1
```

Comment:
This is an unexpected change. Not sure we want it. But it might be the price we pay to do the upgrade incrementally.